projects/kit/components/input-range/test/input-range.component.spec.ts

```diff
@@ -149,7 +149,7 @@ describe('InputRange', () => {
         });
 
         it('does not rounds to nearest multiple of [quantum] until text field losses focus', () => {
-            inputPORight.sendText('8');
+            inputPORight.sendText('8 лет');
```

Comment:
perhaps this is due to the removal of the initial calibration plugin from `input-number`, everything seems to be ok in the demo